docs/docs/guides/smart_contracts/writing_contracts/how_to_prove_history.md

```diff
@@ -5,11 +5,13 @@ sidebar_position: 4
 
 The Aztec Protocol uses an append-only Merkle tree to store hashes of the headers of all previous blocks in the chain as its leaves. This is known as an archive tree. You can learn more about how it works in the [concepts section](../../../aztec/concepts/storage/trees/index.md#archive-tree).
 
-View the History lib reference [here](../../../reference/smart_contract_reference/history_lib_reference.md).
+This page is a quick introductory guide to creating historical proofs proofs from the archive tree.
 
-# History library
+For a reference, go [here](https://docs.aztec.network/reference/smart_contract_reference/aztec-nr/aztec/history/contract_inclusion).
 
-The history library allows you to prove any of the following at a given block height before the current height:
+# Inclusion and non-inclusion proofs
+
+Inclusion and non-inclusion proofs are available in the header of a block. You can prove any of the following at a given block height before the current height:
```

Comment:
```suggestion
Inclusion and non-inclusion proofs refer to proving the inclusion (or absence) of a specific piece of information within a specific Aztec block with a block header. You can prove any of the following at a given block height before the current height:
```

Reply:
The proofs aren't technically available in the header, but the headers are used to create a proof.